static/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```

Comment:
Mostly copied from Via 3, but with the experimental rewriter box removed check box